app/index.js

```diff
@@ -1,7 +1,7 @@
 require('./insights').setup()
 require('log-timestamp')
-const { createMIReport } = require('./reports')
 const { initialise } = require('./storage')
+const { createMIReport } = require('./reports')
```

Comment:
Any particular reason why this line was moved?

Reply:
`initialise` creates the connection to storage which is required before the report generates. Thought it would be easier to follow if the requires followed the usage below

Reply:
Yep, I would agree